book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] According to the project requirement, kindly implement a method to create a new rental using the Rentals class.

```` 
def add_rental (person, date)
    Rental.new(date, self, person)
  end
````